src/lib/init.ts

```diff
@@ -124,16 +124,20 @@ export const init = async (options: InitArg = {}): Promise<FluenceConfig> => {
             })),
         );
 
-  if (
-    existsSync(projectPath) &&
-    (await stat(projectPath)).isDirectory() &&
-    (await readdir(projectPath)).length > 0
-  ) {
-    return commandObj.error(
-      `Directory ${color.yellow(
-        projectPath,
-      )} is not empty. Please, init in an empty directory.`,
-    );
+  if (existsSync(projectPath) && (await stat(projectPath)).isDirectory()) {
+    const directoryContent = await readdir(projectPath);
+    const isNonEmptyDir = directoryContent.length > 0;
+
+    const dirHasOnlyGitInside =
+      directoryContent.length === 1 && directoryContent[0] === ".git";
+
+    if (isNonEmptyDir && !dirHasOnlyGitInside) {
+      return commandObj.error(
+        `Directory ${color.yellow(
+          projectPath,
+        )} is not empty. Please, init in an empty directory.`,
+      );
```

Comment:
wdyt about making it an option?

Like, if `--not-empty` passed, or a question `Directory is not empty, might override existing project. Still continue? Y/n` is answered `yes` – generate project in that directory.